### ISSUE TITLE
About track_comments views, there's a small bug make comment count incorrect.

### ISSUE THREAD

**Issue by rthel** [vitorfs/bootcamp#113]
In template `partial_feed_comments.html` when feed has no comment, it will return one `hint`:
```
<li class="empty">{% trans 'Be the first one to comment' %}</li>
```
 `static/js/feeds.js` line 264 will treat this `hint` as one comment and update comment count to 1, that's not correct.
```
264    $(".comment-count", post_container).text($("ol li", container).length);
```

so how about remove that `hint` or write it in `partial_feed.html` below the `<ol class="prefix"></ol>` , then update `views.trakc_coments` and  use jQuery controll it's visibility.

**Comment by sebastian-code** [comment]
Hi @rthel I haven't been able to replicate the behavior. Until now, the results are consistent with the expected. Can you provide more information?

**Comment by rthel** [comment]
[bug replicate](https://github.com/rthel/huimu/blob/master/bug.jpg)

hello, look at this, I make a jpeg, please click the link and it may help me to explain the issue.

**Comment by sebastian-code** [comment]
It was hard to spot the bug @rthel and now I see what you're talking about. Can you provide a fix for it in a new PR?

**Comment by rthel** [comment]
@sebastian-code Yes, I'd love to : )

**Comment by sebastian-code** [comment]
Solved with #115
